app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java

```diff
@@ -355,7 +361,9 @@ public void boundCameraToRoute() {
       if (bboxPoints.size() > 1) {
         try {
           LatLngBounds bounds = new LatLngBounds.Builder().includes(bboxPoints).build();
-          animateCameraBbox(bounds, CAMERA_ANIMATION_DURATION, new int[] {50, 500, 50, 335});
+          // left, top, right, bottom
+          int topPadding = launchBtnFrame.getHeight() * 2;
```

Comment:
Here too?

Reply:
Yeah this is just to account for the top padding so the route doesn't show up under the launch button